packages/expo-cli/src/commands/upgrade.ts

```diff
@@ -219,6 +219,42 @@ async function upgradeAsync(requestedSdkVersion: string | null, options: Options
     }
   }
 
+  // Check if we can, and probably should, upgrade the (ios) simulator
+  if (Simulator.isPlatformSupported()) {
+    let answer = await prompt({
+      type: 'confirm',
+      name: 'upgradeSimulator',
+      message: 'You might have to upgrade your iOS simulator. Do you want to upgrade it now?',
+    });
+
+    if (answer.upgradeSimulator) {
+      let result = await Simulator.upgradeExpoAsync();
+      if (!result) {
+        log.error('The upgrade of your simulator didn\'t go as planned. You might have to reinstall it manually with expo client:install:ios.');
+      }
+    }
+
+    log.newLine();
+  }
+
+  // Check if we can, and probably should, upgrade the android client
+  if (Android.isPlatformSupported()) {
```

Comment:
Should we check on `Android.assertDeviceReadyAsync()` here too and just log "remember, your Android client might need some love too"? When I tried it errored out because no Android device was connected. It might be a bit too tempting to just say "yes" for a developer.

Reply:
yeah this one is a bit tricky.. maybe we default for both of these to upgrading later? so the `yes` people can get through it quickly. we also might want to tell people that they need to plug in their android device or open their emulator

Reply:
Sorry for the delay, I set the default to false and add a bit more info when you could perform the step. It's still a prompt, mostly to make developers "more-aware" of this. Personally, I think it's the best compromise between forcing them do to something but also protect them from skipping everything 😄 

![Screenshot 2019-11-19 at 00 22 58](https://user-images.githubusercontent.com/1203991/69102443-c1a35100-0a62-11ea-9fa7-e1468ba1bbea.png)